Peer.Domain/Models/PeerPullRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Peer.Domain.Models
+{
+    public class PeerPullRequest
+    {
+        private string _title;
+        private object _assignee;
+
+        public PeerPullRequest(string Title, object Assignee)
```

Comment:
params should be lowercase (the .editorconfig should cause vs to yell at you)

Reply:
Done